javascript/ql/src/Security/CWE-116/examples/IncompleteHtmlAttributeSanitization.js

```diff
@@ -0,0 +1,9 @@
+var app = require('express')();
+
+app.get('/user/:id', function(req, res) {
+	let id = req.params.id;
+	id = id.replace(/<|>/g, ""); // BAD
+	let userHtml = `<div data-id="${id}">${getUserName(id)} || Unknown name</div>`;
```

Comment:
```suggestion
	let userHtml = `<div data-id="${id}">${getUserName(id) || "Unknown name"}</div>`;
```
The resulting HTML being e.g: `<div data-id="123">Erik || Unknown name</div>` looks weird.